system/jlib/jptree.cpp

```diff
@@ -7283,35 +7283,34 @@ class CJSONReader : public CJSONReaderBase<X>, implements IPTreeReader
     {
         offset_t startOffset = curOffset;
         StringBuffer value;
-        if (readValue(value)==elementTypeNull)
```

Comment:
NB: change is to avoid returning on elementTypeNull and continue to beginNode code at bottom,
but still skip child aspects,

Most of diff is indentation/scope